tests/automated_tests/runtime_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+package_name=$(jq ".packages[0].name" .velocitas.json | sed -e 's/^"//' -e 's/"$//')
+package_version=$(jq ".packages[0].version" .velocitas.json | sed -e 's/^"//' -e 's/"$//')
+chosen_runtime=$@
+
+cp /workspaces/vehicle-app-python-template/.velocitas.json "/home/vscode/.velocitas/packages/$package_name/$package_version/.velocitas.json"
```

Comment:
why do you need to copy the json?

Reply:
Because of the way how tests are implemented in decent-runtimes

Reply:
but for that the path is not correct.

Reply:
why path is not correct?

Reply:
please check where the integration-tests of the runtimes are taking the velocitas.json from

Reply:
I did that already

Reply:
but than you would have noticed that all runtimes have their own velocitas.json in the respective test-folders and no velocitas.json in the root of the package